pkg/cache/cache.go

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2020 by PACE Telematics GmbH. All rights reserved.
+// Created at 2020/08/12 by Marius Neugebauer
+
+package cache
+
+import (
+	"context"
+	"time"
+)
+
+// Strings caches strings.
+type Strings interface {
+	// SetString stores the value under the key. Any existing value is
+	// overwritten. If forgetAfter is given, the cache automatically forgets the
+	// value after the duration. If forgetAfter is zero then it is never
+	// automatically forgotten.
+	SetString(ctx context.Context, key, value string, forgetAfter time.Duration) error
```

Comment:
why not use commonly used terminology like `TTL` https://en.wikipedia.org/wiki/Time_to_live ?

Reply:
good idea

Reply:
Done.